server/chat-commands/admin.ts

```diff
@@ -929,8 +929,15 @@ export const commands: ChatCommands = {
 	async eval(target, room, user, connection) {
 		if (!this.canUseConsole()) return false;
 		if (!this.runBroadcast(true)) return;
+		const logRoom = Rooms.get('development');
 
-		if (!this.broadcasting) this.sendReply(`||>> ${target}`);
+		const show = this.message.startsWith('>>') || this.message.startsWith('!eval');
```

Comment:
```suggestion
		const broadcasting = this.broadcasting || this.message.startsWith('>>');
```

(rename other uses of `show` to `broadcasting`.)